ansible/roles/eessi/tasks/main.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install CVMFS repo
+  yum:
+    name: https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+    # NOTE: Can't find any docs on obtaining gpg key
+    disable_gpg_check: true
+- name: Install CVMFS
+  yum:
+    name: cvmfs
+- name: Install EESSI CVMFS config
+  yum:
+    name: https://github.com/EESSI/filesystem-layer/releases/download/latest/cvmfs-config-eessi-latest.noarch.rpm
+    # NOTE: Can't find any docs on obtaining gpg key
+    disable_gpg_check: true
+- name: Add base CVMFS config
```

Comment:
From previous experience with the openhpc role this will be painful - someone will instantly want to configure something. I suggest we make it more flexible, maybe use the `community.general.ini_file` module to dump a dict with some indirection via a _default and _extra dict to allow easily adding and overriding things (we use this pattern elsewhere), maybe something like:
```yaml
cvmfs_config: "{{ cvmfs_config_default | combine(cvmfs_config_extra) }}"

#where:
cvmfs_config_default:
  CVMFS_CLIENT_PROFILE: single
  CVMFS_QUOTA_LIMIT: "{{ cvmfs_quota_limit_mb }}"
cvmfs_config_extra: {}
```

Reply:
Or maybe its an _overrides given it uses combine(), your call.